operations/companies.md

```diff
@@ -164,6 +168,8 @@ Returns all company profiles of the enterprise, possibly filtered by identifiers
 | `Department` | string | optional | The internal segmentation of a company, e.g. sales department. |
 | `DunsNumber` | string | optional | The Dun & Bradstreet unique 9-digit DUNS number. |
 | `CreditRating` | [Credit rating](#credit-rating) | optional | Credit rating to define creditworthiness of the company. |
+| `ReferenceIdentifier` | string | optional | External system identifier - custom identifier used by an external system such as an external database. |
```

Comment:
@Samuel8000 what is the difference between ExternalIdentifier and the ReferenceIdentifier. I see that it's required in the system.

Reply:
@vitezp @Samuel8000 I believe I know the answer, although it would be nice to confirm it 100%. I believe `ExternalIdentifier` was created by the Portfolio team as a global ID for above-enterprise level. They've added it to multiple domain entities, including `Company`, because these all need to be linked and referenced at the portfolio level. The name is a little misleading, because they are not external in the sense of being external to Mews. `ReferenceIdentifier` however came out of a different requirement, specifically from NCH, in this case it really _is_ an external identifier, because it is an ID used by an external database. The obvious name for this field is `ExternalIdentifier`, but this name couldn't be used because it was already taken by the Portfolio team!

Reply:
That's interesting, so could they be somehow consolidated? I'm not sure we need to have externalId that would be diffferent to referenceId, @Samuel8000 please confirm.